### ISSUE TITLE
Maintained?

### ISSUE THREAD

**Issue by Grantherum** [jimmyjon711/FactorioExtended-Plus-Weaponry#27]
Is the FactorioExtended repositories being managed any further?  There are some minor bugs I've fixed on my end I'm willing to push up but if they aren't being maintained any further by Jimmy then what's the point?  Let me know... I don't really want to make a fork because I don't figure I'm going to be maintaining it much better than what it is presently.

**Comment by Quix0r** [comment]
Yes, all of his FE+ mods needs some love. At least they still work but here and there seem to be some inconsistencies. Unfortunately I don't know LUA and have little time left.
